mmselfsup/models/utils/transformer_blocks.py

```diff
@@ -532,3 +534,101 @@ def forward(self,
         x = self.proj_drop(x)
 
         return x
+
+
+class PatchAggregation(BaseModule):
```

Comment:
Why not construct these self attention layers in the neck, instead of creating a new module